src/modules/addTask.js

```diff
@@ -0,0 +1,47 @@
+import Tasks from './classTasks.js';
+import checkDuplicate from './checkDup.js';
+import optionIconUrl from '../imgs/optionsIcon.svg';
+import trashIcon from '../imgs/trashIcon.svg';
+import getData from './arrangeIndex.js';
+
+const input = document.getElementById('task-input');
+const inputItem = document.getElementById('input-item');
+
+getData();
+let id;
+id = Tasks.tasks.length;
```

Comment:
- [ ] The index of each task should be set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5). To do this your first task should have an index of 1, not 0 kindly fix this.